system/ee/legacy/libraries/Functions.php

```diff
@@ -415,7 +415,7 @@ public function determine_error_return()
      * @param string
      * @return void
      */
-    public function redirect($location, $method = false, $status_code = null)
+    public function redirect($location, $method = false, $status_code = null, $forward_url_parameters = false)
```

Comment:
Added as a new parameter here because we only want this setting to apply to redirect functionality when called from a template, not all redirects in the CMS (e.g. from the CP).